.circleci/config.yml

```diff
@@ -42,6 +42,12 @@ jobs:
     steps:
       - checkout
       - <<: *unit
+  "ruby-2.7":
+    docker:
+      - image: circleci/ruby:2.7.0
```

Comment:
Not sure we want to specify exactly 2.7.0. We used to skip TEENY version, so that if Circle updates their image, we don't need to update the code to test against the latest version.

Reply:
Circleci has no 2.7 tag. The only tag they have is 2.7.0. I checked.

Reply:
Thanks for checking!